resources/build/trigger-builds.inc.sh

```diff
@@ -66,6 +66,12 @@ function triggerJenkinsBuild() {
     FORCE=", \"force\": true"
   fi
 
+  local TAG=""
+  # This will only be true if we created and pushed a tag
+  if [ "$action" == "commit" ]; then
```

Comment:
This appears to have broken the build: https://build.palaso.org/viewLog.html?buildId=245552&buildTypeId=Keyman_Test&tab=buildLog&_focus=138

Reply:
`$action` looks to be unbound

Reply:
fixed